pandas_gbq/gbq.py

```diff
@@ -585,6 +585,7 @@ def _bqschema_to_nullsafe_dtypes(schema_fields):
         "DATETIME": "datetime64[ns]",
         "FLOAT": np.dtype(float),
         "GEOMETRY": "object",
+        "INTEGER": "Int64",
```

Comment:
As per comment above, it seems we need to update `docs/source/reading.rst` as well?
(for INTEGER and BOOLEAN)